content/en/user-manual/graphics/lighting/runtime-lightmaps.md

```diff
@@ -25,6 +25,10 @@ The advantages are so:
 
 The disadvantage of using runtime lightmap generation is that currently we do not support baking global illumination, ambient occlusion or some of the other advanced features of a specialized baking tools.
 
+<div class="alert-info">
```

Comment:
I'd change both runtime lightmaps and batching to something along: The use of batching is not compatible with the runtime lightmaps due to each lightmapped object requiring its own unique lightmap.

Reply:
Why does having a unique lightmap affect batching though?

Reply:
well it's the same as having two meshes, and each uses different diffuse texture. They cannot be batches, as the resulting batch can only use single diffuse texture. If they're on an atlas, then can be batched as they'd use the same texture (the atlas one)

Reply:
Gotcah, thanks!